core/api/pod-api/src/default.ts

```diff
@@ -102,7 +102,7 @@ export class DefaultPod implements Pod {
                 else return fs.readFile(path, options);
             }
 
-            readdir(path: string): Promise<string[]> {
+            readdir(path: string): Promise<{ [key: string]: string }[]> {
```

Comment:
(Please read :arrow_down: first) You're changing the default API here, but even if this is TS, it does not mind if the code below is returning what it returns. Again, this will pass through to the `fs` object, which is in the case of the test the `memfs` used as mock, which will return, well, whatever it returned to start with.